pkg/autoops/domain/auto_ops_rule.go

```diff
@@ -74,19 +99,34 @@ func NewAutoOpsRule(
 
 }
 
+func (a *AutoOpsRule) SetStopped() {
+	a.SetAutoOpsStatus(proto.AutoOpsStatus_STOPPED)
+}
+
 func (a *AutoOpsRule) SetDeleted() {
 	a.AutoOpsRule.Deleted = true
 	a.AutoOpsRule.UpdatedAt = time.Now().Unix()
 }
 
+// TODO: Remove this function after auto ops migration.
+// Deprecated
 func (a *AutoOpsRule) SetTriggeredAt() {
 	now := time.Now().Unix()
 	a.AutoOpsRule.TriggeredAt = now
+	a.AutoOpsStatus = proto.AutoOpsStatus_FINISHED
 	a.AutoOpsRule.UpdatedAt = now
 }
 
+func (a *AutoOpsRule) SetFinished() {
+	a.SetAutoOpsStatus(proto.AutoOpsStatus_FINISHED)
+}
+
 func (a *AutoOpsRule) AlreadyTriggered() bool {
-	return a.TriggeredAt > 0
+	return a.TriggeredAt > 0 || a.AutoOpsStatus == proto.AutoOpsStatus_FINISHED
+}
+
+func (a *AutoOpsRule) IsNotFinished() bool {
+	return a.AutoOpsStatus == proto.AutoOpsStatus_WAITING || a.AutoOpsStatus == proto.AutoOpsStatus_RUNNING
```

Comment:
```suggestion
func (a *AutoOpsRule) IsFinished() bool {
	return a.AutoOpsStatus == proto.AutoOpsStatus_FINISHED
```

Reply:
I Fixed it in https://github.com/bucketeer-io/bucketeer/pull/994/commits/96ce3cd3f525cb2d1160037037c6cb9c4a0dfbb1.